oscduplicator/gui/transmitter_container.py

```diff
@@ -0,0 +1,75 @@
+import flet as ft
+
+
+class TransmitterContainer(ft.UserControl):
+    """
+    GUIのうちTransmitterの設定部分
+
+    attribute
+    ---------
+    transmittter_settings: list
+        転送設定, [[port, name, enabled]]
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.transmitter_settings: list = []
+
+    def build(self):
+        return ft.Container(
+            width=600,
+            bgcolor=ft.colors.LIGHT_BLUE_50,
+            padding=20,
+            content=ft.Column(
+                controls=[
+                    ft.Text(value="転送, size=16"),
+                    ft.DataTable(
+                        columns=[
+                            ft.DataColumn(ft.Text("port")),
+                            ft.DataColumn(ft.Text("名前")),
+                            ft.DataColumn(ft.Text("有効化")),
+                            ft.DataColumn(ft.Text("")),
+                        ],
+                        rows=self.data_table_rows(),
+                    ),
+                    ft.ElevatedButton(
+                        text="追加",
+                        icon=ft.icons.ADD,
+                        width=440,
+                        # on_click=
+                    ),
+                ]
+            ),
+        )
+
+    def data_table_rows(self) -> list:
+        """
+        self.transmitter_settingsから,
+        DataTableに表示するためのDataRowのリストを作成する
+        """
+
+        def cell(port="", name="", enabled=False):
+            return ft.DataRow(
+                cells=[
+                    ft.DataCell(ft.Text(value=port)),
+                    ft.DataCell(ft.Text(value=name)),
+                    ft.DataCell(ft.Checkbox(value=enabled)),
+                    ft.DataCell(
+                        ft.ElevatedButton(
+                            text="編集",
+                            # on_click= partialやクロージャを
+                            # 使った関数である必要があるかも?
+                        )
+                    ),
+                ]
+            )
+
+        if not self.transmitter_settings:
+            return [cell()]
+
+        else:
+            return [
+                cell(setting[0], setting[1], setting[2])
```

Comment:
```suggestion
        def create_row(port="", name="", enabled=False):
            return ft.DataRow(
                cells=[
                    ft.DataCell(ft.Text(value=port)),
                    ft.DataCell(ft.Text(value=name)),
                    ft.DataCell(ft.Checkbox(value=enabled)),
                    ft.DataCell(
                        ft.ElevatedButton(
                            text="編集",
                            # on_click= partialやクロージャを
                            # 使った関数である必要があるかも?
                        )
                    ),
                ]
            )

        if not self.transmitter_settings:
            return [create_row()]
        else:
            return [
                create_row(setting[0], setting[1], setting[2])
```

1行を作ってるっぽい名前にすると良さそう